cluster/ubuntu/deployAddons.sh

```diff
@@ -16,19 +16,37 @@
 
 # deploy the add-on services after the cluster is available
 
-set -e
```

Comment:
I removed the `-e` because it makes the following line fail if previous execution of the other function already created the namespace. 

```
${KUBECTL} create -f namespace.yaml 
```

I thought that it is against the idempotency of the script.